pkg/skaffold/runner/dev.go

```diff
@@ -112,13 +113,30 @@ func (r *SkaffoldRunner) doDev(ctx context.Context, out io.Writer, logger *kuber
 		// TODO(modali): Add skipTest boolean to Tester itself to avoid this check.
 		if !r.runCtx.SkipTests() {
 			if err = r.Test(ctx, out, bRes); err != nil {
```

Comment:
`r.Test` is being called twice (line 115 and `128`). Is there a reason for this?

Reply:
This is legacy code. Since there was no explicit testing phase before, Tests were being run explicitly after every build. I cleaned up the calls.